compute/list_all_instances.go

```diff
@@ -35,12 +36,17 @@ func listAllInstances(w io.Writer, projectID string) error {
 	}
 	defer instancesClient.Close()
 
+	// Use the `MaxResults` parameter to limit the number of results that the API returns per response page.
```

Comment:
Nothing wrong with changes here, but would you update L31. Looks like an incorrect sample value.

Reply:
updated